common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -294,7 +294,7 @@ siege_works_01 = {
 	
 	character_modifier = {
```

Comment:
These icons are missing.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103485033-cc5aaa00-4e0c-11eb-93c4-c4d3ab729445.png)

</details>

Reply:
fixed icons not showing up, but poor resalution